pkg/estimator/hll/hll.go

```diff
@@ -241,6 +242,10 @@ func (h *Plus) MarshalBinary() (data []byte, err error) {
 		return nil, nil
 	}
 
+	if h.sparse {
+		h.mergeSparse()
+	}
```

Comment:
This change is so that the serialised format of the HLL sketch is repeatable. I could have also made the `set` serialisation repeatable by sorting but this was easier.